src/components/table/_table.scss

```
@@ -226,8 +226,12 @@
     max-height: 0;
   }
 
+  99% {
+    max-height: 100vh;
+  }
+
   100% {
-    max-height: 1000px;
+    max-height: max-content;
```

While not as clever, changing this to `unset`, essentially just removing any applied `max-height` works across Chrome, Firefox and Safart
```suggestion
    max-height: unset;
```

---

Codesandbox to test: https://codesandbox.io/s/nested-table-bug-forked-k0l5v